sdk/core/System.Memory.Data/src/System.Memory.Data.csproj

```diff
@@ -1,8 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>Contains the BinaryData type, which is useful for converting between string, streams, JSON, and bytes.</Description>
+    <Description>Contains the BinaryData type, which is useful for converting between strings, streams, JSON, and bytes.</Description>
```

Comment:
@weshaggard @chidozieononiwu why does this change appear as part of this PR?

Reply:
That is a good question. @chidozieononiwu looks like your recent change always does an amend see https://github.com/Azure/azure-sdk-for-net/pull/16528/. That is the bug here.